be/src/exprs/agg/hll_sketch.h

```diff
@@ -23,23 +23,30 @@
 
 namespace starrocks {
 
+struct HLLSketchState {
+    std::unique_ptr<DataSketchesHll> hll_sketch = nullptr;
+};
+
 /**
  * RETURN_TYPE: TYPE_BIGINT
  * ARGS_TYPE: ALL TYPE
  * SERIALIZED_TYPE: TYPE_VARCHAR
  */
 template <LogicalType LT, typename T = RunTimeCppType<LT>>
 class HllSketchAggregateFunction final
-        : public AggregateFunctionBatchHelper<DataSketchesHll, HllSketchAggregateFunction<LT, T>> {
+        : public AggregateFunctionBatchHelper<HLLSketchState, HllSketchAggregateFunction<LT, T>> {
 public:
     using ColumnType = RunTimeColumnType<LT>;
 
     void reset(FunctionContext* ctx, const Columns& args, AggDataPtr state) const override {
```

Comment:
is there some test cases with regarding to window?

Reply:
Like `appox_count_distinct`, `ds_hll_count_distinct` don't support as `analytic function` which may be meaningless. It's forbidden in the FE:
```
          // ds_hll_count_distinct(col)
            addBuiltin(AggregateFunction.createBuiltin(DS_HLL_COUNT_DISTINCT,
                    Lists.newArrayList(t), Type.BIGINT, Type.VARBINARY,
                    true, false, true));
            // ds_hll_count_distinct(col, log_k)
            addBuiltin(AggregateFunction.createBuiltin(DS_HLL_COUNT_DISTINCT,
                    Lists.newArrayList(t, Type.INT), Type.BIGINT, Type.VARBINARY,
                    true, false, true));
            // ds_hll_count_distinct(col, log_k, tgt_type)
            addBuiltin(AggregateFunction.createBuiltin(DS_HLL_COUNT_DISTINCT,
                    Lists.newArrayList(t, Type.INT, Type.VARCHAR), Type.BIGINT, Type.VARBINARY,
                    true, false, true));
```